darwin/importer/importer.py

```diff
@@ -377,5 +440,15 @@ def _import_annotations(
     payload: Dict[str, Any] = {"annotations": serialized_annotations}
     if append:
         payload["overwrite"] = "false"
+    elif delete_for_empty and not annotations:
```

Comment:
True, but I wanted to make it explicit. Like "no matter what you do here, if this 2 things happen, overwrite must be true".

Reply:
To me personally, this is just confusing. In this function, as this point, neither `delete_for_empty` nor annotations being empty matter. This function is and should be as simple as "if append, then false, otherwise always true". 
Otherwise one could think we have some logic to correctly handle the case of `delete_for_empty and not annotations` while we don't - we expect those to be handled earlier by this function execution of lack there of.

Reply:
Done!